README.md

```diff
@@ -1,2 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portfolio
-My portfolio that shows my educational background, projects and technical skills.
+
+> This is a mobile-first portfolio project which is built to show projects, background and the ways to get in touch with the developer.
+
+## Built With
+
+- Major languages: HTML & CSS
+- Frameworks: N/A
+- Technologies used: Git & VScode
+
+## Live Demo (if available)
+
+N/A
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- clone this repo to your local machine.
+- open it with a text editor (VScode)
+
+## Authors
+
+- GitHub: [@khawanin](https://github.com/Khawanin)
+- LinkedIn: [@khawaninali](https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Kindly make sure you fix this broken link([`MIT](./LICENSE)`) as once it is clicked it shows an error page. You fix it by only adding the **[LICENSE](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** file in your root directory as you have already provided a **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**.